bootstrap/components/topology/storm-topology-component.json

```diff
@@ -55,6 +55,40 @@
             "defaultValue": "hdfs://localhost:9000/tmp/hbase"
           }
         ]
+      },
+      {
+        "uiName": "Clusters Security Config",
+        "fieldName": "clustersSecurityConfig",
+        "tooltip": "Clusters Security Configuration",
+        "isOptional": true,
+        "type": "array.object",
+        "defaultValue": null,
+        "fields": [
+          {
+            "uiName": "Cluster Name",
```

Comment:
@HeartSaVioR yes, we can change `"clusterName"` type from "string" to "enum" and pass the names to options, but I am wondering whether this cluster names option will be pre-defined into this JSON file or will be coming via some API's depending on cluster and environment?